scripts/build-windows.ps1

```diff
@@ -95,25 +95,26 @@ if ($env:CLEAN -eq "true")
 # Ensure that the system dependencies are at the correct version - fail if not
 # Check MSVS Tools through MSVS_VERSION
 $EXPECTED_MSVST_VERSION = @("2019","2022")
-$MSVS_REG_PATH = "Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\Setup"
+
+$MSVS_REG_PATH = "Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\X64"
```

Comment:
This last part of the path might not work for arm machines.

Reply:
how can we test it?

Reply:
maybe ask if some of the contributors has an arm machine running windows?

Reply:
or maybe we just put a `TODO` comment stating this (above line 99) and merge this PR. If/when we get someone with an arm machine AND using windows, then we can revisit?

Reply:
Also, is this version specific? Does this work for 2019 and 2022 build tools ?